types/activation.go

```diff
@@ -38,6 +39,7 @@ type ActivationTxHeader struct {
 
 type NIPSTChallenge struct {
 	NodeId         NodeId
+	Coinbase       address.Address
```

Comment:
There is no reason for the coinbase account to be part of the NIPSTChallenge, can you move it to the ATX itself?